### DISCUSSION TITLE
Add support for Baidu Qianfan's big model

### DISCUSSION THREAD

**Discussion by whto** [langgenius/dify#3424] [Suggestion]
### Self Checks

- [X] I have searched for existing issues [search for existing issues](https://github.com/langgenius/dify/issues), including closed ones.
- [X] I confirm that I am using English to submit this report (我已阅读并同意 [Language Policy](https://github.com/langgenius/dify/issues/1542)).
- [X] Pleas do not modify this template :) and fill in all the required fields.

### 1. Is this request related to a challenge you're experiencing?

yes

### 2. Describe the feature you'd like to see

Can reference the Qianfan model, such as Llama-2-13B-Chat, Qianfan-Chinese-Llama-2-13B

### 3. How will this feature improve your workflow or experience?

Using a Baidu account allows access to more models.

### 5. Can you help us with this feature?

- [ ] I am interested in contributing to this feature.

**Comment by Air-duct** [comment]
praise
